src/GraphQL/packages.config

```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Antlr4" version="4.5-alpha002" targetFramework="net45" userInstalled="true" />
```

Comment:
Any reason why you were using the alpha version here in the first place @joemcbride? It seems to be running fine with the stable version on my end.